### PR TITLE
Add Grafana panels with workspace traces metrics

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Add several panels to Grafana to observe Workspace API traces:
Screenshot from 2019-09-10 14-04-36

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13502

### CODE REVIEW

**Comment by skabashnyuk** [comment]
@mkuznyetsov I would like to ask you for a couple of adjustments.
 - minimum values on panels. For the timers, it makes sense to set it to ```0```
 - add units of measurements
 - for all workspace start/stop related measurements in case of aggregation function  (like ```rate```) please use time window ```[1h]```
 - On a new panel please use ```Null as zeoro``` configuration parameter

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 1]
```diff
@@ -4190,31 +4815,33 @@ data:
           "bars": false,
           "dashLength": 10,
           "dashes": false,
-          "datasource": "$datasource",
+          "datasource": "che",
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: I doubt that we need this change

**Review by metlos — COMMENTED** [review]
Do we plan to also add these to the helm chart?

If yes, then some of the charts need to be modified, because they're openshift specific (which makes sense in the openshift template ;) ).

Otherwise LGTM.

**Comment by skabashnyuk** [comment]
>Do we plan to also add these to the helm chart?

Not at this moment.

**Review by skabashnyuk — APPROVED** [review]

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 2]
```diff
@@ -3931,49 +3935,619 @@ data:
...
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "rate(che_workspace_start_time_seconds_sum[5m])/rate(che_workspace_start_time_seconds_count [5m])",
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: 1h

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml [hunk 3]
```diff
@@ -3931,49 +3935,619 @@ data:
...
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "rate(che_workspace_stop_time_seconds_sum[5m])/rate(che_workspace_stop_time_seconds_count [5m])",
```
Comment (skabashnyuk) [inline deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: 1h

File: deploy/openshift/templates/monitoring/grafana-dashboards.yaml (hunk 1 above)
Comment (mkuznyetsov) [reply deploy/openshift/templates/monitoring/grafana-dashboards.yaml]: fixed

**Pull request merged**